ochk/structures_virtual_networks.go

```diff
@@ -0,0 +1,28 @@
+package ochk
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func flattenStringSlice(in []string) *schema.Set {
+	out := &schema.Set{
+		F: schema.HashString,
+	}
+
+	for _, v := range in {
+		out.Add(v)
+	}
+	return out
+}
+
+func flattenUUIDs(in []strfmt.UUID) *schema.Set {
```

Comment:
to już chyba nie używane?

Reply:
zostawmy to jeszcze chwilę na wypadek jak się znów pojawi w backendach uuid?